### PR TITLE
Update textfield

### PR DESCRIPTION
**Pull request by abubuwe**
# Description

Following the work to update the dropdown field, this resurrects the work to update the textfield. It adds the background, updates the error icon and prevents the field from spilling over the screen.

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

image

## Checklist

- [ ] I have updated mockServer
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

Update the color pallette, especially feedbackBad
Convert `ValidatedTextInput` into a FC.

### CODE REVIEW

File: src/components/GenericTextField.tsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import { KeyboardTypeOptions, StyleSheet } from 'react-native';
 import { FieldWrapper } from './Screen';
```
Comment (selvach) [inline src/components/GenericTextField.tsx:6]: Could you remove unused imports on Line 2

File: src/features/patient/fields/HeightQuestion.tsx [hunk 2]
```diff
@@ -36,93 +38,91 @@ export class HeightQuestion extends Component<Props, object> {
...
-                  returnKeyType="next"
-                  onSubmitEditing={() => {}}
-                  keyboardType="numeric"
-                />
-              </View>
-              <View style={styles.tertiaryField}>
+      <FieldWrapper style={styles.fieldWrapper}>
+        <RegularText>{i18n.t('your-height')}</RegularText>
+        {isUSCountry() ? (
+          <View style={styles.primaryFieldRow}>
```
Comment (selvach) [inline src/features/patient/fields/HeightQuestion.tsx:44]: There's a refactor here. Lines 44-69 are a repeated fragmented at Line 73-98

File: src/components/GenericTextField.tsx (hunk 1 above)
Comment (abubuwe) [reply src/components/GenericTextField.tsx:6]: Thanks. Will do across files.

File: src/features/patient/fields/HeightQuestion.tsx (hunk 2 above)
Comment (abubuwe) [reply src/features/patient/fields/HeightQuestion.tsx]: Refactored! Felt nice. Thanks :)

Comment (abubuwe) [reply src/features/patient/fields/HeightQuestion.tsx:43]: Moving refactor into a separate PR.

**Pull request merged**